### ISSUE TITLE
bug: noice.nvim "breaks" vim-easy-align

### ISSUE THREAD

**Issue by lervag** [folke/noice.nvim#569]
### Did you check docs and existing issues?

- [X] I have read all the noice.nvim docs
- [X] I have searched the existing issues of noice.nvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.10.0-dev-633+g8a788e2daa

### Operating system/version

Arch Linux

### Describe the bug

[vim-easy-align](https://github.com/junegunn/vim-easy-align) is a great plugin that provides very convenient commands for aligning content. However, it provides a user interface that relies on `:echohl` and `:echon`. The result can be seen in the following screenshot:

![image](https://github.com/folke/noice.nvim/assets/998898/38cfd308-bc59-434d-8c1d-32ef81a28951)

Without Noice, the same "state" would look like this:

![image](https://github.com/folke/noice.nvim/assets/998898/caf41255-32fd-45ba-b504-72addb59567f)

### Steps To Reproduce

1. Install vim-easy-align and noice.nvim
2. Start the `:LiveEasyAlign` command and observe

### Expected Behavior

I know from #532 that noice.nvim does not really work well with `:echon` and user interfaces that relies on updating the message area with `:echo`s. Still, I think it would be good with either just a brief confirmation that there is nothing to do here, or perhaps a consideration of possible solutions.

I'm thinking about a specific possible solution where noice.nvim provides some sort of wrapper over a mapping and/or command that temporarily reenables the default message area and sets `cmdheight` back to 1.

If you are interested in investigating possible solutions for this topic, then I would be more than willing to help e.g. by providing good steps to reproduce and a minimal setup.

**Comment by lervag** [comment]
I realize that I might get what I want simply with `:Noice enable` and `:Noice disable`. I've created a custom vim-easy-align wrapper that uses these commands and it seems to work as expected, along the lines of:

```vim
function wrapper(...)
  Noice disable
  set cmdheight=1
  " ...
  set cmdheight=0
  Noice enable
endfunction
```

This might suffice as a solution.

**Comment by lervag** [comment]
Ok, it seems to work well. I created a (relatively) simple [wrapper for vim-easy-align](https://github.com/lervag/dotnvim/blob/73432a5/plugin/easy-align.vim) that temporarily disables noice.nvim. [Here is the full changeset](https://github.com/lervag/dotnvim/commit/73432a5302041e7701ffce3c5820a57c967b201e) I applied to my configuration, in case others may find it useful.

I believe this resolves my issue, but I'll leave it open in case this sparks thoughts or ideas from you or in case you should have further suggestions on

**Comment by folke** [comment]
yeah, plugins using echo for any kind of formatting is tricky. Glad you found a work-around
